docs/copied-from-beats/shared-securing-beat.asciidoc

```diff
@@ -0,0 +1,38 @@
+[id="securing-{beatname_lc}"]
```

Comment:
Since security is an advanced topic and not the thing that you typically try first, I'd move the whole "Securing APM Server" section towards the end; maybe just before the exported fields.

Reply:
Agreed it makes sense a bit later, once you have the basics of configuration set - moved it down a bit.